cubedash/templates/dashboard.html

```diff
@@ -0,0 +1,82 @@
+{% extends "layout/base.html" %}
+
+{% block title %}Audit Dashboard{% endblock %}
+{% block head %}
+    {{ super() }}
+    <style type="text/css">
+        .sub-header {
+            line-height: 1.4em;
+        }
+
+        a {
+            font-weight: bold;
+        }
+
+        .panel {
+            line-height: 1.5em;
+        }
+        th, td{
+            padding: 5px 10px;
+        }
+        body {
+            width: 100%;
+            max-width: 100%;
+        }
+        thead tr {
+            display: block;
+        }
+        tbody {
+            display: block;
+            height: 700px;
+            overflow: auto;
+        }
+        table th {
+            width: 220px;
```

Comment:
Putting a width on the headers but not body is causing them not to align:
![Screenshot from 2020-09-23 11-05-42](https://user-images.githubusercontent.com/25688/93952728-eb1cbf80-fd8c-11ea-8edf-60924cb483c8.png)

I'd rather no width as the columns have very different content.

Numbers columns are good to right-align for readability.

(If easy, maybe reuse style from the other table-based results pages, eg: https://explorer.dea.ga.gov.au/region/ls7_level1_scene/91_80/2007/8 )

Reply:
this is a flow on affect to enable table header always visible on the page.